test/membrane_mp3_mad/decoder_test.exs

```diff
@@ -99,4 +106,18 @@ defmodule Membrane.MP3.MAD.DecoderTest do
     assert byte_size(new_state.queue) == 4
     assert new_state.native == ctx.state.native
   end
+
+  @tag :manual
+  test "manual test using Membrane.PortAudio.Sink" do
```

Comment:
couldn't we compare with a fixture instead?

Reply:
I tried, but I was unable to generate same output as our decoder produces

Reply:
Let's use the output from our decoder then :P